robot_specific_config/abb_irb1200_5_90_moveit_config/config/joint_limits.yaml

```diff
@@ -5,30 +5,30 @@ joint_limits:
   joint_1:
     has_velocity_limits: true
     max_velocity: 5.027
-    has_acceleration_limits: false
-    max_acceleration: 0
+    has_acceleration_limits: true
+    max_acceleration: 50.0
```

Comment:
moveit will refuse running if no acceleration limit is set, I put a random value here but should probably double check it